code/_helpers/game.dm

```diff
@@ -184,6 +184,18 @@
 
 	return hear
 
+/proc/get_turfs_in_view(var/R, var/atom/source)
+	var/turf/start_point = get_turf(source)
+	var/list/hear = list()
+
+	if(!start_point)
+		return hear
+
+	var/list/range = hear(R, start_point)
```

Comment:
`hear` is declared as a list in this function. What you're doing here is. Weird? You're not calling the function `hear()`, that's for sure.